module_utils/certificate_lsr/providers/PROVIDERS.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
```

Comment:
Why should we use this file instead of `init`? What linter issue is triggered if we use `init`?

Reply:
Ansible requires empty `__init__.py` files in module_utils directories

Reply:
https://docs.ansible.com/ansible/2.9/dev_guide/testing/sanity/empty-init.html

Reply:
Thank you, @richm.